Leave/leave_interface.py

```diff
@@ -57,20 +56,24 @@ async def HandleLeaveReactions(client, payload):
     if status == None:
         return
 
-    if utils.isNotBot(payload.member) and utils.IsAdmin(payload.member) and leave_db.IsLeaveRequestPending(payload.message_id):
+    if not (utils.isNotBot(payload.member) and utils.IsAdmin(payload.member)):
+        return
+        
+    if status == "Reverted":
```

Comment:
This variable would be better named action instead of status I think

Reply:
Done